packages/kicad-diff-git/kicad-diff-git.pacscript

```diff
@@ -0,0 +1,31 @@
+name="kicad-diff-git"
+repology=("project: kicad-diff-git")
+epoch="1"
+homepage="https://github.com/leoheck/KiCad-Diff"
+build_depends=("git" "python3-pip")
+depends=("python3")
+gives="kicad-diff"
+description=" Scripts for performing image diffs between pcbnew layout revisions Resources"
+maintainer="Marcel Robitaille <mail@marcelrobitaille.me>"
+arch=("any")
+provides=("kicad-diff")
+conflicts=("kicad-diff")
+url="https://github.com/leoheck/KiCad-Diff.git"
+
```

Comment:
```suggestion
```

Reply:
Isn't `url` required? It's used in `pkgver()` and to download the source code, right? If I try to remove it, I get:

```
(kicad-diff-git) Do you want to view/edit the pacscript [y/N]
[+] INFO: Sourcing pacscript
fatal: no path specified; see 'git help pull' for valid url syntax
fatal: no path specified; see 'git help pull' for valid url syntax
[!] ERROR: Package does not contain version
```

Reply:
I said to to remove the blank line, not URL.

Reply:
Oh sorry the diff in GitHub makes it look like I should replace 11-14 with just an empty line